lib/src/main/java/vmath/Mat4.java

```diff
@@ -0,0 +1,14 @@
+package vmath;
+
+public class Mat4 {
+    private final float[] m;
+
+    public Mat4() {
+        m = new float[] {
+                1.0f, 0.0f, 0.0f, 0.0f,
+                0.0f, 1.0f, 0.0f, 0.0f,
+                0.0f, 0.0f, 1.0f, 0.0f,
+                0.0f, 0.0f, 0.0f, 1.0f
```

Comment:
I understand that the default indentation in IntelliJ IDEA is set like that, but personally, I find it inconsistent. I would prefer a use of 4 spaces everywhere. Upon reviewing the selected lines here, it seems that 8 spaces have been used.

Reply:
Thank you for pointing that out. I've now modified the code to have uniform 4-space indentation